packages/kbn-test/src/functional_tests/lib/run_elasticsearch.ts

```diff
@@ -184,8 +184,18 @@ function getESServerlessOptions(
   const serverlessHost: string | undefined =
     config.has('esServerlessOptions.host') && config.get('esServerlessOptions.host');
 
+  const kbnServerArgs =
+    (config.has('kbnTestServer.serverArgs') &&
+      (config.get('kbnTestServer.serverArgs') as string[])) ||
+    [];
+  const serverlessArg = kbnServerArgs.find((arg) => arg.startsWith('--serverless'));
```

Comment:
Is it guaranteed that this argument will be passed in the `--arg=value` format and not `--arg value`?

Reply:
Good point, Alex. I believe it can be both, so I made a change in a7f037647fc5e8cbeb3dcab4a3a05ce98b998817